internal/controller/controllers/conditions.go

```diff
@@ -67,6 +67,22 @@ const (
 	ClusterValidationsUnknownMsg string = "The cluster's validations have not yet been calculated"
 	ClusterValidationsFailingMsg string = "The cluster's validations are failing:"
 
+	ClusterFailedCondition string = hivev1.ClusterInstallFailed
+	ClusterFailedReason    string = "InstallationFailed"
+	ClusterFailedMsg       string = "The installation failed:"
+	ClusterNotFailedReason string = "InstallationNotFailed"
+	ClusterNotFailedMsg    string = "The installation has not failed"
+
+	ClusterStoppedCondition       string = hivev1.ClusterInstallStopped
+	ClusterStoppedFailedReason    string = "InstallationFailed"
+	ClusterStoppedFailedMsg       string = "The installation has stopped due to error"
+	ClusterStoppedCanceledReason  string = "InstallationCancelled"
```

Comment:
nit: stick with canceled or cancelled for keeping convention. can be tackled later on.